docs/sphinx/users/micromanager/index.txt

```diff
@@ -0,0 +1,39 @@
+Micro-Manager
```

Comment:
Similar to the other `user` pages, is it possible to add a one-level sentence with a link to the website explaining what Micro-manager is?